### PR TITLE
Bumping hello eks anywhere version

### PR DESCRIPTION
**Pull request by jonahjon**
Signed-off-by: jonahjon <jonahjones094@gmail.com>

*Issue #, if available:*

*Description of changes:*

Testing cross region replication.

### CODE REVIEW

File: generatebundlefile/data/input_121_prod.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ packages:
         repository: hello-eks-anywhere
         registry: public.ecr.aws/eks-anywhere
         versions:
-            - name: 0.1.1-bc7dc6bb874632972cd92a2bca429a846f7aa785
+            - name: 0.1.1-083e68edbbc62ca0228a5669e89e4d3da99ff73b
```
Comment (ivyostosh) [inline generatebundlefile/data/input_121_prod.yaml:11]:
```suggestion
            - name: 0.1.1-083e68edbbc62ca0228a5669e89e4d3da99ff73b
```

File: generatebundlefile/data/input_122_prod.yaml [hunk 2]
```diff
@@ -8,7 +8,7 @@ packages:
         repository: hello-eks-anywhere
         registry: public.ecr.aws/eks-anywhere
         versions:
-            - name: 0.1.1-bc7dc6bb874632972cd92a2bca429a846f7aa785
+            - name: 0.1.1-083e68edbbc62ca0228a5669e89e4d3da99ff73b
```
Comment (ivyostosh) [inline generatebundlefile/data/input_122_prod.yaml:11]:
```suggestion
            - name: 0.1.1-083e68edbbc62ca0228a5669e89e4d3da99ff73b
```

File: generatebundlefile/data/input_123_prod.yaml [hunk 3]
```diff
@@ -8,7 +8,7 @@ packages:
         repository: hello-eks-anywhere
         registry: public.ecr.aws/eks-anywhere
         versions:
-            - name: 0.1.1-bc7dc6bb874632972cd92a2bca429a846f7aa785
+            - name: 0.1.1-083e68edbbc62ca0228a5669e89e4d3da99ff73b
```
Comment (ivyostosh) [inline generatebundlefile/data/input_123_prod.yaml:11]:
```suggestion
            - name: 0.1.1-083e68edbbc62ca0228a5669e89e4d3da99ff73b
```

**Review by vincentni — COMMENTED** [review]
image
shouldn't the version be `0.1.1-0566a131e10d6aed7cae86e5882f9da51f941f90`?

**Review by ivyostosh — COMMENTED** [review]
Looks good. I just removed the trailing white spaces

**Review by vincentni — APPROVED** [review]
/lgtm
/approve

**Pull request merged**